generative_agents/demo/penpal_app.py

```diff
@@ -1,105 +0,0 @@
-import os
-import openai
-import streamlit as st
-from streamlit_chat import message
-
-from generative_agents.modeling.agent_helpers import load_llm, load_pinecone, load_agent
-from generative_agents.utility import logger, well_known_paths
-from generative_agents.utility.utility import load_params
-from generative_agents.modeling.interactions import talk_to_penpal, start_conversation_w_penpal
-
-log = logger.init("Streamlit app")
-
-
-def generate_response(user_input):
-    res = talk_to_penpal(agent, user_input, params)
-    return res
-
-
-st.title("EF PenPal 🤖 ")
```

Comment:
With this file renaming, may I please request an update of our README to have
`streamlit run penpal_app/app.py`

Reply:
Done!